Src/Support/Google.Apis.Auth/OAuth2/IGoogleCredential.cs

```diff
@@ -0,0 +1,41 @@
+﻿/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+namespace Google.Apis.Auth.OAuth2
+{
+    /// <summary>
+    /// Represents a Google credential. Defines functionality that
```

Comment:
Maybe refer to "types that can be used as an underlying credential for GoogleCredential"?

Reply:
Done